File: leetcode-solutions/0455-assign-cookies/solution.java
class Solution {
    public int findContentChildren(int[] g, int[] s) {
        Arrays.sort(g);
        Arrays.sort(s);
        int i=0;
        for(int cookie:s){
            if(i<g.length && cookie>=g[i]){
                i+=1;
            }
        }
        return i;
    }
}

File: leetcode-solutions/0455-assign-cookies/solution.py
class Solution(object):
    def findContentChildren(self, g, s):
        """
        :type g: List[int]
        :type s: List[int]
        :rtype: int
        """
        g.sort()
        s.sort()
        i=0
        for cookie in s:
            if (i<len(g) and cookie>=g[i]):
                i+=1
        return i
        
